### PR TITLE
Fix Android Studio warnings in BrickDataManager

### LINKED ISSUE #189: Android Studio warnings should be fixed in BrickDataManager

### PR DESCRIPTION
**Pull request by thusson13**
Fixes https://github.com/wayfair/brickkit-android/issues/189

### CODE REVIEW

File: BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java [hunk 1]
```diff
@@ -94,21 +93,20 @@ private void setRecyclerView(@NonNull RecyclerView recyclerView, int orientation
...
+        this.recyclerView.addItemDecoration(new BrickRecyclerItemDecoration(this));
         this.recyclerView.setItemAnimator(new AvoidFlickerItemAnimator());
 
         GridLayoutManager gridLayoutManager = new WFGridLayoutManager(recyclerView.getContext(), orientation);
         gridLayoutManager.setSpanSizeLookup(new BrickSpanSizeLookup(recyclerView.getContext(), this));
         recyclerView.setLayoutManager(gridLayoutManager);
 
-        LinkedList<BaseBrick> items = getRecyclerViewItems();
-        int itemCount = items.size();
+        LinkedList<BaseBrick> recyclerViewItems = getRecyclerViewItems();
```
Comment (kenyee) [inline BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java]: can this just be a plain List on the left hand side?

File: BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java [hunk 2]
```diff
@@ -608,10 +600,10 @@ public void removeItem(BaseBrick item) {
 
             if (brickRecyclerAdapter != null) {
                 brickRecyclerAdapter.safeNotifyItemRemoved(index);
-                LinkedList<BaseBrick> items = getRecyclerViewItems();
+                LinkedList<BaseBrick> recyclerViewItems = getRecyclerViewItems();
```
Comment (kenyee) [inline BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java]: can this be List<>?

File: BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java (hunk 1 above)
Comment (thusson13) [reply BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java]: Yeah, I was gonna do more of the LinkedList -> List API changes all at once

File: BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java (hunk 2 above)
Comment (thusson13) [reply BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java]: Yeah, I was gonna do more of the LinkedList -> List API changes all at once

File: BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java (hunk 1 above)
Comment (thusson13) [reply BrickKit/bricks/src/main/java/com/wayfair/brickkit/BrickDataManager.java]: Did the internal ones

**Review by kenyee — APPROVED** [review]

**Pull request merged**